### ISSUE TITLE
Question: Rest API

### ISSUE THREAD

**Issue by eltNEG** [moluwole/Bast#17]
Does this package support responding with Json? I can't find anything on it in the documentation and example.

**Comment by moluwole** [comment]
Yes it does. In your controller, using `self.json({message: 'response'})`. This sends a json response and automatically adds the `Content-Header: application/json` to the response header

**Comment by moluwole** [comment]
Remember your response must be a `dict`
